socorro/tests/external/es/test_supersearch.py

```diff
@@ -458,7 +491,8 @@ def test_get_with_bool_operators(self, es_helper):
 
         assert resp["total"] == 2
         assert len(resp["hits"]) == 2
-        assert not resp["hits"][0]["accessibility"]
+        for hit in resp["hits"]:
+            assert not hit["accessibility"]
```

Comment:
this test was flaky because it was only checking one hit, checking every hit made it reliably fail until I restored correct behavior.